hera_qm/xrfi.py

```diff
@@ -1540,6 +1540,12 @@ def xrfi_run(ocalfits_files=None, acalfits_files=None, model_files=None, data_fi
         (should have a .uvh5 at the end). Output products will replace extension
         with various output labels. For example, output_prefixes='filename.uvh5'
         will result in products with names like 'filename.cross_flags1.h5'.
+    throw_away_edges : bool, optional
```

Comment:
My only suggestion was to expand this doc string, so I just did it. If you're OK with it, then feel free to merge.